### DISCUSSION TITLE
how to attack a custom model that isn't a neural network

### DISCUSSION THREAD

**Discussion by teenaxta** [Trusted-AI/adversarial-robustness-toolbox#2092] [Q&A] [answered]
I am trying to see how attacks work or how noise is added in various attacks. For that I have a made a simple class:

`class quadrant():
    def predict(x):
        if x < 0:
            return 0
        else:
            return 1
            
new_model = quadrant()`

Issue is that every attack asks for an estimator built using one of ARTs estimators. I dont want to give it a neural network. I just want to see how a 1D point oscillates

**Comment by beat-buesser** [comment, marked as answer]
Hi @teenaxta For such applications you could use ART's `BlackBoxClassifier` estimator and provide it your function `predict` as input.
